libs/librepcb/libraryeditor/lib/libraryoverviewwidget.cpp

```diff
@@ -135,6 +143,20 @@ bool LibraryOverviewWidget::save() noexcept {
   }
 }
 
+void LibraryOverviewWidget::symShowContextMenu(const QPoint &pos) noexcept {
+    QPoint globalPos = mUi->lstSym->mapToGlobal(pos);
+    QMenu menu;
+    menu.addAction(QIcon(":/img/actions/delete.png"), tr("Remove"), this, &LibraryOverviewWidget::symRemove);
```

Comment:
I wouldn't use a separate function and signal/slot connection for `QAction` handlers. I think it's simpler to get the return value of `QMenu::exec()` to decide what action was triggered. Then you can also use [`QListWidget::itemAt()`](http://doc.qt.io/qt-5/qlistwidget.html#itemAt) to get the item which was actually right-clicked. Later we need it anyway to decide which context menu entries should be shown and which not...

See for example `SymbolEditorState_Select::openContextMenuAtPos()`.